setup.cfg

```diff
@@ -38,7 +38,7 @@ install_requires =
     backpack-for-pytorch
     asdfghjkl
 # Require a specific Python version, e.g. Python 2.7 or >= 3.4
-python_requires = >=3.8
+python_requires = >=3.7
```

Comment:
Any particular reason for the downgrade?

Reply:
Many people I talked to still have TF v1.x as a dependency for various reasons. The last TF v1.x doesn't support Py > 3.7.